tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+"""Pytest Config and shared fixtures."""
+import pathlib
+
+import pytest
+
+
+@pytest.fixture()
+def feature_file(tmp_path):
+    basic_feature = tmp_path / "basic.feature"
+    test_dir = pathlib.Path(__file__).parent
+    with open(test_dir / "basic.feature") as feature_fo:
+        basic_feature.write_text(feature_fo.read())
```

Comment:
Not sure if that is the same as, or subtly different from, https://docs.python.org/3/library/shutil.html#shutil.copyfile

Reply:
Question about `shutil.copyfile` still stands...

Reply:
both of these were lift-and-shift, so I didn't look carefully, and missed the comment(s) in the larger rewrite, but I just tested and. `shutil.copyfile` works fine here (and below), so I'll make that change.